docker/docker-compose.yml

```diff
@@ -1,12 +1,11 @@
-version: '3'
 services:
   mysql_writer:
     user: root
     build:
       context: ./writer
       dockerfile: Dockerfile
     ports:
-      - 33306:3306
+      - "33306:3306"
```

Comment:
v3 이후로 docker compose 파일에서 포트포워딩을 명시할 때는 문자열로 명시하는 것이 표준입니다!
yaml 파서가 `xx:yy` 형태의 base60 으로 숫자를 파싱하기 때문에, 단순히 `x:y` 형태의 정수로 지정할 경우 60 이하의 포트에서 문제가 발생할 수 있어요.

> HOST:CONTAINER should always be specified as a (quoted) string, to avoid conflicts with [yaml base-60 float](https://yaml.org/type/float.html).

ref: https://docs.docker.com/reference/compose-file/services/#ports